java/util/src/main/java/com/google/protobuf/util/JsonFormat.java

```diff
@@ -151,9 +153,26 @@ public Printer includingDefaultValueFields() {
      * current {@link Printer}.
      */
     public Printer preservingProtoFieldNames() {
-      return new Printer(registry, includingDefaultValueFields, true);
+      return new Printer(registry, includingDefaultValueFields, true, omittingInsignificantWhitespace);
     }
-    
+
+    /**
+     * Create a new  {@link Printer}  that will omit all insignificant whitespace in the JSON output. This new Printer clones all other configurations from the current Printer.
```

Comment:
This line is too long. The column limit is 100 according to [Google Java Style Guide](https://google.github.io/styleguide/javaguide.html#s4.4-column-limit).